### ISSUE TITLE
Proposal to redesignate None as recombinant lineage X*

### ISSUE THREAD

**Issue by tsekizuka** [cov-lineages/pango-designation#263]
Lineage renaming proposal: None -> X* (XB or XC)

by Tsuyoshi Sekizuka

Description

Sub-lineage of: None
Earliest sequence: 2021-08-12
Most recent sequence: 2021-09-01
Countries circulating: Japan

Notable genome recombination between Delta (AY.29) and Alpha (B.1.1.7) variants in SARS-CoV-2 have been identified in clinical specimens in Japan.

Genomes

Virus name,Accession ID,Collection date,Location
hCoV-19/Japan/PG-107020/2021,EPI_ISL_4882472,2021-08-13,Asia / Japan / Tokyo
hCoV-19/Japan/PG-113440/2021,EPI_ISL_4882473,2021-09-01,Asia / Japan / Tokyo
hCoV-19/Japan/PG-122116/2021,EPI_ISL_4882474,2021-08-13,Asia / Japan / Tokyo
hCoV-19/Japan/PG-126853/2021,EPI_ISL_4882475,2021-08-25,Asia / Japan / Kanagawa
hCoV-19/Japan/PG-128229/2021,EPI_ISL_4882476,2021-08-18,Asia / Japan / Tokyo
hCoV-19/Japan/PG-133737/2021,EPI_ISL_4882477,2021-08-12,Asia / Japan / Tokyo

Evidence

<img width="923" alt="スクリーンショット 2021-10-18 13 35 57" src="https://user-images.githubusercontent.com/34445991/137673192-478e05e7-b72a-4c6e-b5d1-5a1857819ca3.png">
<img width="922" alt="スクリーンショット 2021-10-18 13 36 06" src="https://user-images.githubusercontent.com/34445991/137673207-ba649632-9069-4e9c-b8c6-5d1764cd8724.png">
<img width="861" alt="スクリーンショット 2021-10-18 13 36 17" src="https://user-images.githubusercontent.com/34445991/137673213-5d11fbc2-b7f7-40b1-b25f-758c7eda3fc3.png">
<img width="1701" alt="スクリーンショット 2021-10-18 13 43 14" src="https://user-images.githubusercontent.com/34445991/137673217-fc3c4395-a54c-4917-8886-9deb08c2ca3a.png">

Proposed lineage name

None -> X*

**Comment by tsekizuka** [comment]
Thanks for your response.

For comparative analysis of recombinant lineage, I uploaded three phylogenic trees with Alpha (B.1.1.7), Delta (AY.29) and recombinant lineage samples.

In the uploaded Nextclade data, PG-48062 (top sample) and PG-58832 (bottom sample) are Delta (AY.29) and Alpha (B.1.1.7), respectively.

All sequences were validated using mapping analysis, these sequences is not artifact.

We have submitted the manuscript of recombinant lineage sample to medrxiv.
https://www.medrxiv.org/content/10.1101/2021.10.11.21264606v1
